tests/refc/prims/Test.idr

```diff
@@ -0,0 +1,24 @@
+module Test
+
+import Data.IOArray
+import Data.IORef
+import System.Info
+
+main : IO ()
+main = do
+  do
```

Comment:
The `do`s here are a bit unusual (nested and used as a visual separator I gather), but I'm not about to turn down a PR based on that especially in a test case.

Reply:
Thanks.

I am not aiming for visual purposes, but to limit the scope and lifetime of the variable (like C's bracket).

I wish there was a more concise and beautiful way to write this.